src/main/java/PositiveCounter.java

```diff
@@ -1,8 +1,10 @@
+import java.util.Random;
+
 /**
  * An always Positive Counter
  */
 public class PositiveCounter extends CRDT {
-    private static final long serialVersionUID = 1L;
+    private static final long serialVersionUID = new Random().nextLong();
```

Comment:
Ah, sorry!  I should have been more specific.  This will definitely break.  You need to set it to a random constant *in the source*, not generate a new random number at runtime.  The important thing is that both sides have the same number, and it's a lot easier to debug when that number is unique to the particular class.